homeassistant/util/dt.py

```diff
@@ -2,10 +2,13 @@
 import datetime as dt
 import re
 
+# pylint: disable=unused-import
+from typing import Any, Union, Optional, Tuple  # NOQA
```

Comment:
typing.Any is used in some type comments of variables.

mypy requires it to be imported but neither pylint nor flake8 detect the usage.